src/weights.jl

```diff
@@ -342,22 +345,6 @@ Compute the weighted sum of an array `v` with weights `w`, optionally over the d
 wsum(v::AbstractVector, w::AbstractVector) = dot(v, w)
 wsum(v::AbstractArray, w::AbstractVector) = dot(vec(v), w)
 
-# Note: the methods for BitArray and SparseMatrixCSC are to avoid ambiguities
```

Comment:
Are you sure these ambiguities no longer exist?

Reply:
The tests pass.

Reply:
But do they cover these types?

Reply:
Lines 33 through 37 of the test file do, don't they?